Source/WebCore/style/values/backgrounds/StyleLineWidth.h

```diff
@@ -43,14 +43,13 @@ struct LineWidth {
     using Length = Style::Length<CSS::NonnegativeUnzoomed>;
 
     Length value;
-    bool isKeyword { false };
 
-    constexpr LineWidth(Length length) : value { length }, isKeyword { false } { }
+    constexpr LineWidth(Length length) : value { length } { }
     constexpr LineWidth(CSS::ValueLiteral<CSS::LengthUnit::Px> literal) : value { literal } { }
 
-    constexpr LineWidth(CSS::Keyword::Thin) : value { 1 }, isKeyword { true } { }
-    constexpr LineWidth(CSS::Keyword::Medium) : value { 3 }, isKeyword { true } { }
-    constexpr LineWidth(CSS::Keyword::Thick) : value { 5 }, isKeyword { true } { }
+    constexpr LineWidth(CSS::Keyword::Thin) : value { 1 } { }
```

Comment:
I think it probably makes sense to remove these constructors taking keywords for now, as a caller is probably doing the wrong thing.

Reply:
I think its wrong, because now when one would use a keyword, zoom should also be applied.